helper_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakeFolder(t *testing.T) {
+	privateK, publicK, _ := MakeFolder("dummy-folder")
+	assert.Equal(t, "dummy-folder/private-key", privateK)
+	assert.Equal(t, "dummy-folder/public-key", publicK)
+
+	defer os.RemoveAll("dummy-folder")
+}
+
+func TestCreateRSAkeys(t *testing.T) {
+	privateK, publicK, _ := MakeFolder("dummy-folder")
+	assert.Nil(t, CreateRSAkeys(privateK, publicK))
+
+	defer os.RemoveAll("dummy-folder")
+}
+
+func TestParsePrivateRSAKey(t *testing.T) {
+	privateK, publicK, _ := MakeFolder("dummy-folder")
```

Comment:
A bit overkill to create a test for a helper file, but it makes for good practice.